### PR TITLE
Update specutils description to better reflect the true goals

### PR DESCRIPTION
**Pull request by eteq**
This updates the README and the docs index page with a more detailed description of the current "vision" for specutils.  

It also removes some material that to me looked like "documentation" from the README.  That's always dangerous because it inevitably becomes out-of-date.  I could add back a link that points to the "first steps" page or something, though...

@keflavich and/or @crawfordsm - look reasonable to you?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,142 +1,68 @@
...
+    <br />
 
-  spec1d = Spectrum1D.from_array(wave, flux)
-  spec1d.wavelength
-  <Quantity [ 6000., 6001., 6002.,...,  8997., 8998., 8999.] Angstrom>
-  spec1d.flux
-  <Quantity [ 0.75639906, 0.23677036, 0.08408417,...,  0.82740303, 0.38345114,
-              0.77815595] W / (Angstrom m2 sr)>
+Specutils is an `Astropy`_ affiliated package with the goal of providing a
+shared set of Python representations of astronomical spectra, and basic tools to
```
Comment (keflavich) [inline README.rst]: no comma

File: README.rst [hunk 2]
```diff
@@ -1,142 +1,68 @@
...
+operate on these spectra.  The effort is also meant to be a "hub", helping to
+unite the Python astronomical spectroscopy community around shared effort, much
+as `Astropy`_ is meant to for the wider astronomy Python ecosystem.
 
-Or you can read a Spectrum from a .fits file with the **read_fits** method:
+Note that Specutils is *not* intended as an all-in-one spectroscopic analysis or
+reduction tool.  While it provides some baseline analysis (following the Python
```
Comment (keflavich) [inline README.rst]: maybe `basic` instead of `baseline` ("baseline" = "continuum" in radio parlance)

Comment (eteq) [inline README.rst]: Hah, never would have thought of that but I see your point.  This is why we need all different kinds of astronomers ;)

**Comment by eteq** [comment]
Alright, addressed @keflavich's comments.  Lets give it a day or two for someone else to chime in but it's ready to merge if there's nothing else.

File: README.rst [hunk 3]
```diff
@@ -1,142 +1,68 @@
...
+operate on these spectra.  The effort is also meant to be a "hub", helping to
+unite the Python astronomical spectroscopy community around shared effort, much
+as `Astropy`_ is meant to for the wider astronomy Python ecosystem.
 
-Or you can read a Spectrum from a .fits file with the **read_fits** method:
+Note that Specutils is *not* intended as an all-in-one spectroscopic analysis or
+reduction tool.  While it provides some basic analysis (following the Python
+philosophy of "batteries included"), it is also meant to facilitate connecting
+together disparate reduction pipelines and analysis tools through shared data
+representations.
```
Comment (crawfordsm) [inline README.rst]: Maybe at this spot, add links to some of the other packages like pyspeckit, spectralcube, specreduce?  I'm okay all with those being added in a separate PR

File: docs/index.rst [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
-******************
-An AstroPy affiliated package containing operations and tools on astronomical spectra.
+*********
+
+Specutils is an `Astropy`_ affiliated package with the goal of providing a
+shared set of Python representations of astronomical spectra and basic tools to
+operate on these spectra.  The effort is also meant to be a "hub", helping to
+unite the Python astronomical spectroscopy community around shared effort, much
+as `Astropy`_ is meant to for the wider astronomy Python ecosystem.
+
```
Comment (crawfordsm) [inline docs/index.rst]: Another possible place to add other packages

**Comment by crawfordsm** [comment]
Overall looks good!  Added two comments but happy to add those later if you don't want to do those now.

**Comment by eteq** [comment]
@crawfordsm - good suggestions, but I think the easiest is for me to merge this and then *you* can add a PR adding those packages, as I think you have a better idea of what goes there than I do!

**Pull request merged**